### PR TITLE
PMM-6774 add kubeconfig view

### PR DESCRIPTION
**Pull request by lunaticusgreen**
[PMM-6774](https://jira.percona.com/browse/PMM-6774)

### CODE REVIEW

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesClusterActions/KubernetesClusterActions.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { FC, useCallback } from 'react';
+import { Messages } from 'pmm-dbaas/DBaaS.messages';
+import { MultipleActions } from 'pmm-dbaas/components/MultipleActions/MultipleActions';
+// import { isClusterChanging } from '../DBCluster.utils';
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesClusterActions/KubernetesClusterActions.tsx:4]: Can be deleted.

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.test.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('KubernetesInventory::', () => {
 
     expect(rows.length).toBe(kubernetesStub.length + 1);
   });
-  it('cancels delete cluster', () => {
+  xit('cancels delete cluster', () => {
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.test.tsx:23]: This should still be working.

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx [hunk 3]
```diff
@@ -20,10 +21,9 @@ export const KubernetesInventory: FC<KubernetesProps> = ({
   loading,
 }) => {
   const styles = useStyles(getStyles);
-  const [kubernetesToDelete, setKubernetesToDelete] = useState<any>({
-    kubernetesClusterName: '',
-  });
+  const [selectedCluster, setSelectedCluster] = useState<any>({ kubernetesClusterName: '' });
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx:24]: Create a type for this if there isn't one.

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx [hunk 4]
```diff
@@ -125,48 +119,30 @@ export const KubernetesInventory: FC<KubernetesProps> = ({
...
+            size="md"
+            onClick={() => setDeleteModalVisible(false)}
+            data-qa="cancel-delete-kubernetes-button"
+          >
+            {Messages.kubernetes.deleteModal.cancel}
+          </Button>
+          <Button
+            variant="destructive"
+            size="md"
+            onClick={() => {
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx:135]: Create a function for this and useCallback

File: pmm-app/src/pmm-dbaas/components/Kubernetes/ViewClusterConfigModal/ViewClusterConfigModal.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  setVisible,
+  selectedCluster,
+}) => {
+  const [kubeconfig, setKubeconfig] = useState('');
+  const [loading, setLoading] = useState(false);
+  const outputRef = useRef<HTMLPreElement>(null);
+
+
+  const copyToClipboard = useCallback(() => {
+    showSuccessNotification({ message: 'Copied' });
```
Comment (tiagomotasantos) [inline pmm-app/src/pmm-dbaas/components/Kubernetes/ViewClusterConfigModal/ViewClusterConfigModal.tsx:25]: Message files.

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: pmm-app/src/pmm-dbaas/components/Kubernetes/ViewClusterConfigModal/ViewClusterConfigModal.tsx (hunk 5 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/ViewClusterConfigModal/ViewClusterConfigModal.tsx:25]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesClusterActions/KubernetesClusterActions.tsx (hunk 1 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesClusterActions/KubernetesClusterActions.tsx:4]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx (hunk 4 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx:135]: done

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.test.tsx (hunk 2 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.test.tsx]: Ooops, forgot to add real tests after copy-paste, added

File: pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx (hunk 3 above)
Comment (lunaticusgreen) [reply pmm-app/src/pmm-dbaas/components/Kubernetes/KubernetesInventory.tsx]: 👍

**Review by tiagomotasantos — APPROVED** [review]

**Pull request merged**